README.rst

```diff
@@ -0,0 +1,518 @@
+libCEED: Efficient Extensible Discretization
+============================================
+
+.. image:: https://github.com/CEED/libCEED/workflows/C/Fortran/badge.svg
+   :alt: GitHub Actions
+   :target: https://github.com/CEED/libCEED/actions
```

Comment:
It's README.md now, so this can be removed.

Reply:
Whoops, not sure how this made it in after the merge from main.  Removed now in  593b5cac93e94ddcc5cbe975940a49cb9548585c